tests/etl/athena_to_dataframe.py

```diff
@@ -0,0 +1,28 @@
+from unittest import TestCase
+
+from hip_data_tools.aws.common import AwsConnectionSettings
+from hip_data_tools.etl.athena_to_dataframe import AthenaToDataFrameSettings, AthenaToDataFrame
+
+
+class TestS3ToDataFrame(TestCase):
+
+    def test_the_etl_should_work(self):
```

Comment:
Is this an integration test? If so we should not let it run in circle ci right ?

Reply:
its not running in circle ci. See file name

Reply:
Good trick